src/lib/utils.ts

```diff
@@ -48,6 +49,45 @@ export function addMODUKFilters(nunjucksEnv: Environment) {
   nunjucksEnv.addFilter('addCustomMODUKClass', addCustomMODUKClass)
 }
 
+/**
+ * Add MOD.UK svg loader to an existing Nunjucks environment.
+ * @param nunjucksEnv Nunjucks environment
+ * Based from https://github.com/11ty/eleventy/blob/v2.0.1/src/Engines/Nunjucks.js#L166-L228
+ * as https://mozilla.github.io/nunjucks/api.html#custom-tags
+ */
+export function addSvgExtension(nunjucksEnv: Environment) {
+  class SvgLoader {
+    tags = ['svg']
+
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    parse(parser: any, nodes: any) {
+      const tok = parser.nextToken()
+
+      const args = parser.parseSignature(true, true)
+
+      // Nunjucks bug with non-paired custom tags bug still exists even
+      // though this issue is closed. Works fine for paired.
+      // https://github.com/mozilla/nunjucks/issues/158
+      if (args.children.length === 0) {
+        args.addChild(new nodes.Literal(0, 0, ''))
+      }
+
+      parser.advanceAfterBlockEnd(tok.value)
+
+      return new nodes.CallExtension(this, 'run', args)
+    }
+    /* eslint-enable @typescript-eslint/no-explicit-any */
+
+    /* eslint-disable class-methods-use-this */
+    run(_context: unknown, fileName: string) {
+      const svgPath = `${join(__dirname, '../assets/svg', basename(fileName, '.svg'))}.svg`
+      return new runtime.SafeString(readFileSync(svgPath, 'utf8'))
+    }
+    /* eslint-enable class-methods-use-this */
+  }
+  nunjucksEnv.addExtension('MODUKSvgLoader', new SvgLoader())
+}
+
```

Comment:
reimplemented the SVG shortcode in the moduk environment, before it was in 11ty but then realised that not going to work for portability. 
https://github.com/11ty/eleventy/blob/v2.0.1/src/Engines/Nunjucks.js#L166-L228
https://mozilla.github.io/nunjucks/api.html#custom-tags

Reply:
Upon reflection I should update the tag name to something less generic like MODUKSvg